system_tests/bigquery_test_data.json

```diff
@@ -0,0 +1,66 @@
+[
```

Comment:
@tswast's examples, derived from the bigquery doc snippets (not ours, the ones used on cloud.google.com).